src/Illuminate/Support/Arr.php

```diff
@@ -510,4 +510,23 @@ public static function where($array, callable $callback)
 
         return $filtered;
     }
+
+    /**
+     * Map one arrays keys to another.
+     *
+     * @param array $array
+     * @param array $mapArray
+     *
+     * @return array
+     */
+    public static function mapOver($array, $mapArray)
```

Comment:
Please don't place a function randomly at the end of the file. Please respect the current alphabetical ordering.